edk2-rockchip/Platform/Firefly/Drivers/LogoDxe/License.txt

```diff
@@ -0,0 +1,2 @@
+The source code in this directory is licensed under the standard EDK2 terms.
+

```

Comment:
Please add a note about the license terms for Logo.bmp

Reply:
Due to the big time difference this document can take a long time, 
but i already sent a request to firefly

Reply:
> Please add a note about the license terms for Logo.bmp

They did agree to allow use of the logo. I am waiting for their license document so we can attach it here if needed.

@jaredmcneill I tried contacting you on twitter @spikerguy and also on your email but no reply. It is regarding Device sample for you.

Reply:
@S199pWa1k9r @spikerguy have you received any feedback from Firefly in this regard?